docs/testspec/src/18_tpm.rst

```diff
@@ -0,0 +1,468 @@
+   +------------------------+-------------------------------------------------------------------------+
+   | **Expected Output:**   | n/a                                                                     |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Steps:**             | #. Create a TPM2 context and an unauthenticated session                 |
+   |                        |                                                                         |
+   |                        | #. Instantiate the passed persistent ECDSA key pair using an incorrect  |
+   |                        |    authentication value                                                 |
+   |                        |                                                                         |
+   |                        | #. Check that the signature creatino fail with a "TPM2 Error"           |
+   +------------------------+-------------------------------------------------------------------------+
+
+
+.. table::
+   :class: longtable
+   :widths: 20 80
+
+   +------------------------+-------------------------------------------------------------------------+
+   | **Test Case No.:**     | TPM-ECDSA-3                                                             |
+   +========================+=========================================================================+
+   | **Type:**              | Positive Test                                                           |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Description:**       | Create a Key Pair, Use it, Make it Persistent, Evict it                 |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Preconditions:**     | The TPM 2.0 software emulator is running and provisioned properly       |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Input Values:**      | - TCTI name and config                                                  |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Expected Output:**   | n/a                                                                     |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Steps:**             | #. Create a TPM2 context and an authenticated session via the Storage   |
+   |                        |    Root Key                                                             |
+   |                        |                                                                         |
+   |                        | #. Create a transient unrestricted key with the auth_value "secret" and |
+   |                        |    the elliptic curve "secp521r1"                                       |
```

Comment:
technically some of the tests listed here are secp192 or 384. Maybe omit the curve?

Reply:
good idea.